vitrina/users/forms.py

```diff
@@ -0,0 +1,156 @@
+from crispy_forms.helper import FormHelper
+from crispy_forms.layout import Layout, Div, Field, Submit, ButtonHolder
+from django.contrib.auth import authenticate
+from django.contrib.auth.forms import PasswordResetForm as BasePasswordResetForm
+from django.core.exceptions import ValidationError
+from django.forms import Form, EmailField, CharField, PasswordInput, ModelForm, BooleanField
+
+from django.utils.translation import gettext_lazy as _
+
+from vitrina.users.models import User
+
+
+class LoginForm(Form):
+    email = EmailField(label=_("El. paštas"), required=True)
+    password = CharField(widget=PasswordInput, label=_("Slaptažodis"), required=True)
+
+    def __init__(self, request=None, *args, **kwargs):
+        self.request = request
+        self.user_cache = None
+        super().__init__(*args, **kwargs)
+        self.helper = FormHelper()
+        self.helper.layout = Layout(
+            Div(Div(Field('email', css_class='input', placeholder=_("El. paštas")),
+                    css_class='control'), css_class='field'),
+            Div(Div(Field('password', css_class='input', placeholder=_("Slaptažodis")),
+                    css_class='control'), css_class='field'),
+            ButtonHolder(Submit('submit', _("Prisijungti"), css_class='button is-primary'))
+        )
+
+    def clean(self):
+        email = self.cleaned_data.get('email')
+        password = self.cleaned_data.get('password')
+
+        if email is not None and password:
+            self.user_cache = authenticate(self.request, email=email, password=password)
+            if self.user_cache is None:
+                raise ValidationError(_("Neteisingi prisijungimo duomenys"))
+
+        return self.cleaned_data
+
+    def get_user(self):
+        return self.user_cache
+
+
+class RegisterForm(ModelForm):
+    first_name = CharField(label=_("Vardas"), required=True, )
+    last_name = CharField(label=_("Pavardė"), required=True)
+    email = EmailField(label=_("El. paštas"), required=True, error_messages={})
+    password = CharField(widget=PasswordInput, label=_("Slaptažodis"), required=True)
```

Comment:
Shouldn't there be a `password2` field, for checking if password was correctly entered two times?

https://docs.djangoproject.com/en/3.2/topics/auth/default/#django.contrib.auth.forms.UserCreationForm